### DISCUSSION TITLE
從ChatGPT到Midjourney，回顧AI生成大爆發的一年！未來還有哪些應用？

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#933] [General]
Midjourney和Stable Diffusion在網路社群掀起了浪潮，讓人人都能成為「藝術家」，但也導致有藝術家利用該工具當輔助，引起各方人士不滿，視覺藝術家與設計師們今年連連受到生成式AI挑戰，使用的核心技術「擴散（Diffusion）」，透過添加雜訊慢慢破壞數據結構，然後從這些被拆解的數據中，透過篩選、降噪與隨機採樣等步驟，就可以獲得一張全新的圖。

強大而快速的機器學習不免會讓人擔憂許多工作、甚至人類藝術的可取代性。不過不管是圖像生成AI與ChatGPT要走上提高專業度、甚至商業化，仍有一段路要走；在科學研究方面，AI作為輔助，也將可以提高研發的效率。

我認為可以以正向的想法去看待這些AI，畢竟現在存在的技術都是成熟發展的產物，AI還有非常大的進步空間，與其想著自己會被取代，不如多學習一點，努力成為可以與AI共存的戰友。

https://www.bnext.com.tw/article/73396/ai-diffusion-2022

**Comment by Evan0979** [comment]
ChatGPT真的很誇張,即使已經出現了,我心裡一時還是有點難以接受

**Comment by StevenWanglolz** [comment]
真的好佩服openAI喔~把人工智慧發揮的淋漓盡致，造福了很多人，但也為特定職業的人帶來了許多挑戰。不過身為學生，openAI真的幫了我大忙!

**Comment by CPCpyt** [comment]
我覺得技術沒有好壞，重要的是人們如何運用。
AI的創作某些程度上也帶給大家很多的啟發，也許能讓我來的藝術推升到新的層次。
